graphql_compiler/scripts/generate_test_sql/__init__.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017-present Kensho Technologies, LLC.
```

Comment:
All the files that are brand new are `Copyright 2018-present`, not `2017-present`. This is very important!